### ISSUE TITLE
Error: Evaluation failed: TypeError: f.MediaPrep.prepRawMedia is not a function

### ISSUE THREAD

**Issue by macleysousa** [wppconnect-team/wa-js#983]
## Description

send file imagem failed

## Environment

- **WhatsApp version: *2.2312.7*
- **WA-JS version(s): *2.20.2*
- **Browser: *Chromium 101.0.4950.0*  
- **OS: *Windows 10* 

## Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

## Log Output

```
Error: Evaluation failed: TypeError: f.MediaPrep.prepRawMedia is not a function
```

## Your Code

```
If relevant, paste all of your challenge code in here
```

## Additional context / Screenshot

Add any other context about the problem here. If applicable, add screenshots to help explain.

**Comment by macleysousa** [comment]
fix v2.22.2

**Comment by edgardmessias** [comment]
Já aproveita e entra no grupo do WPPConnect: https://chat.whatsapp.com/LJaQu6ZyNvnBPNAVRbX00K
